pkgs/development/tools/continuous-integration/drone/default.nix

```diff
@@ -2,7 +2,7 @@
 
 buildGoModule rec {
   name = "drone.io-${version}";
-  version = "1.6.5";
+  version = "1.9.0";
```

Comment:
The sha256 in the src attribute and probably the vendorSha256 need to be updated as well.

Reply:
I'm suprised too: Neither of them. I faked both and they resulted the same.

Reply:
It should be 
```diff
-  vendorSha256 = "1dvf8vz3jr9smki3jql0kvd8z8rwdq93y7blbr2yjjfsdvx6lxl1";
+  vendorSha256 = "0idf11sr417lxcjryplgb87affr6lgzxazzlyvk0y40hp8zbhwsx";

   src = fetchFromGitHub {
     owner = "drone";
     repo = "drone";
     rev = "v${version}";
-    sha256 = "05cgd72qyss836fby0adhrm5p8g7639psk2yslhg6pmz0cqfbq9m";
+    sha256 = "1lsyd245fr1f74rpccvvw41h5g75b79afrb8g589bj13ggjav0xy";
   };
```

Reply:
Done. That's still _darknix_ to me. Why https://github.com/NixOS/nixpkgs/pull/93681#issuecomment-662732113, then?

Reply:
Nix picks up existing (cached) versions of the `src` and the `vendor` go module dependencies based on the hash from the nix store. It won't notice if e.g. that cached source contains a different version of drone than intended.